modules/react/tabs/lib/TabsList.tsx

```diff
@@ -17,7 +17,7 @@ import {
 import {useTabsModel} from './useTabsModel';
 
 // Use `Partial` here to make `spacing` optional
-export interface TabListProps<T = unknown> extends Partial<ExtractProps<typeof Stack, never>> {
+export interface TabListProps extends Omit<Partial<ExtractProps<typeof Stack, never>>, 'children'> {
```

Comment:
I guess here we're giving children a different type which makes sense to omit! Nice!